lib/Sema/TypeCheckConcurrency.h

```diff
@@ -560,6 +560,13 @@ bool isAccessibleAcrossActors(
     const DeclContext *fromDC,
     llvm::Optional<ReferencedActor> actorInstance = llvm::None);
 
+/// Determines if the 'let' can be read from anywhere within the given module,
+/// regardless of the isolation or async-ness of the context in which
+/// the var is read.
+bool isLetAccessibleAnywhere(const ModuleDecl *fromModule,
+                             VarDecl *let,
+                             ActorReferenceResult::Options &options);
```

Comment:
Note that I added this as a new overload because the original is in `include/swift/AST/ActorIsolation.h` and I didn't see a good reason to expose `ActorReferenceResult` outside of Sema.